xapian-applications/omega/templates/query

```diff
@@ -81,23 +85,30 @@ $if{$not{$msizeexact},
   All $nice{$msize} matches,
   $nice{$add{$topdoc,1}}$if{$ne{$add{$topdoc,1},$last},-$nice{$last}} of exactly $nice{$msize} matches}
 }
+</section>
 <hr>
 </center>
+
+<section>
 $list{$map{$queryterms,$list{$html{$uniq{$unstem{$_}}},<b>,</b>/<b>,</b>}:&nbsp;$nice{$freq{$_}}},Term frequencies: ,$. ,}
 <br><small>Search took $time seconds</small>
+</section>
 <table>
 $hitlist{<tr><td valign=top>
 ${<img src="/icons/omega/score-$div{$percentage,10}.png" alt="$percentage%" height=16 width=32>}
+
 <div title="$percentage%" style='float:left;width:60px;height:10px;border:1px solid black;margin-top:4px;'>
 <div style='width:$div{$mul{$percentage,6},10}px; height:10px; background-color: red;'>
 </div></div>
 <div style='float:left;margin-top:2px;font-size:x-small;'>
+<time>
 $if{$and{$field{modtime},$ne{$field{modtime},-1}},
 <script type="text/javascript">write_date($add{$field{modtime}});</script>
 <noscript>
 <span title="$html{$date{$field{modtime},%Y-%m-%d %H:%M:%S}}">
 Modified:<br><b>$html{$date{$field{modtime},%Y-%m-%d}}</b></span>
 </noscript><br>
```

Comment:
html5 is a bit wishy-washy here; I'd only use `hr` in narrative text, like you get in novels within a chapter when some time has passed or something. Where we're moving between sections of the document, we should be able to style off existing elements.

(Note that my comment referred to a `br` not an `hr`.)